java/sample-apps/aws-sdk/README.md

```diff
@@ -24,6 +24,8 @@ terraform init
 terraform apply
 ```
 
+For the agent version, to optionally enable a pipeline to send Prometheus metrics to a Prometheus backend, provide the ARN of a Lambda layer with a custom collector configuration in a file named `config.yaml` when prompted after running the `terraform apply` command. 
```

Comment:
Ah I don't think we need to talk about Prometheus either - the custom config file could be anything, perhaps enabling the `logging` exporter for example. So just `to change the configuration of the OpenTelemetry collector, you can...`

Reply:
changed